packages/aws-cdk/lib/api/deployments/deployments.ts

```diff
@@ -297,23 +293,13 @@ interface AssetOptions {
 }
 
 export interface BuildStackAssetsOptions extends AssetOptions {
-  /**
-   * Options to pass on to `buildAssets()` function
-   */
-  readonly buildOptions?: BuildAssetsOptions;
```

Comment:
Only used by `buildSingleAsset`. None of the call sites specify `buildOptions`. Not used in `buildSingleAsset` at all.